docs/contributing/contributing-to-airbyte/updating-documentation/README.md

```diff
@@ -2,29 +2,27 @@
 
 Our documentation uses [GitBook](https://gitbook.com) and all the [Markdown](https://guides.github.com/features/mastering-markdown/) files are stored in our Github repository.
 
-There are 3 different ways you can update the documentation.
+## Modify on GitHub
```

Comment:
Seems like there are identical files `README.md` as well as a `updating-documentation.md` -- comments apply to both :)

Reply:
weird
I'll see how to fix it